### DISCUSSION TITLE
I am probably stupid or something... (New item)

### DISCUSSION THREAD

**Discussion by abrefael** [TomBursch/kitchenowl#437] [Q&A]
Love the program.
Built something similar myself, but not as beautiful.
The only thing is I can't figure out how to add new items to the "Items" list.
Found two ways, extremely convoluted(!):
1. New Recipe -> Optional ingredients -> "searching" for anything that wasn't there earlier...
2. Importing a json file evey time I want to add a new item to the list
Are these the only ways?
Is there no "New Item" button?
And while we're at it, is there a way to add additional csv's for other icons?

**Comment by TomBursch** [comment]
You can search for items in the items view, and it should always suggest the literal term you searched as an item. By clicking on it, the item is added to the database and to the shopping list.
